code/DebugBar.php

```diff
@@ -201,7 +201,7 @@ public static function includeRequirements()
         }
         // If jQuery is already included, set to false
         $js = Requirements::backend()->getJavascript();
-        foreach ($js as $url) {
+        foreach ($js as $url => $args) {
             $name = basename($url);
```

Comment:
Won't this break if it's a numeric assoc array? e.g. `[0 => 'myfile.css', 1 => 'myfile.js']`

I believe a sanity check e.g.
```
if (is_array($args)) {
    $name = $url;
} else {
   $name = $args;
}
```
would make more sens